### ISSUE TITLE
Make ARPACK available as solver in SLEPc

### ISSUE THREAD

**Issue by jvellojin** [conda-forge/slepc-feedstock#81]
### Comment:

Dear all.

As the title says, is it possible to include the ARPACK solver to the installation of SLEPc in Anaconda?. In particular, I would like to use it in a conda environment for solving eigenvalue problems. Maybe some FEM softwares like Fenics or Ngsolve could benefit from this in another virtual environments such as Docker.

Currently, when trying something like Eps.setType(Eps.Type.ARPACK) results in the error:

```File "slepc4py/SLEPc/EPS.pyx", line 335, in slepc4py.SLEPc.EPS.setType
petsc4py.PETSc.Error: error code 86
[0] EPSSetType() at /Users/runner/miniforge3/conda-bld/slepc_1705997419779/work/src/eps/interface/epsbasic.c:172
[0] Unknown type. Check for miss-spelling or missing package: https://petsc.org/release/install/install/#external-packages
[0] Unknown EPS type given: arpack
```

even though I have `arpack`  installed from [conda-forge](https://github.com/conda-forge/arpack-feedstock)

My system is an M2 MacBook Pro.

Thanks in advance.

**Comment by joseeroman** [comment]
Why do you want to use ARPACK? SLEPc's default eigensolver (Krylov-Schur) is mathematically equivalent to ARPACK's method, but it is generally more efficient and in some cases more robust (as ARPACK may miss eigenvalues in certain situations).

**Comment by dalcinl** [comment]
@jvellojin If you really want ARPACK after what Jose said, maybe you should submit a pull request with the required changes? My wild guess is that it is a trivial change:  a couple lines adding the arpack  package as a dependency in meta.yml, and a `--with-arpack=1` argument in the configure invocation.

**Comment by jvellojin** [comment]
I totally agree with @joseeroman. I was suggesting this just to have the additional option which is available by default in SLEPc on Debian based systems. I'm not familiarized with pull request for conda packages yet, so I will leave this as it is.
